test-configs.yaml

```diff
@@ -21,6 +21,12 @@ file_system_types:
     arch_map:
       armhf: {arch: arm}
 
+  debian_stretchv4l2:
+    url: 'http://staging-storage.kernelci.org/images/rootfs/debian/stretchv4l2/20181109.0'
```

Comment:
Please update the URL to use `20181116.0` as it has the latest `v4l2-parser.sh` script.

PS: I know this is annoying, see discussion on IRC about having a "latest rootfs version" symlink on the storage server...

Reply:
I have updated the link to use the images built today.